cclient/BUILD

```diff
@@ -1,9 +1,9 @@
 cc_library(
     name = "api",
     srcs = [
-        "iota_api.c",
+        "iota_client_core_api.c",
```

Comment:
can it be only `iota_client_api`?

Reply:
`iota_client_api` is first name pop up in my mind, but considering extended API, I think `iota_client_core_api` for core APIs and `iota_client_extended_api` for extended.